.gitignore

```diff
@@ -12,3 +12,23 @@ makelog.txt
 # Unknown
 /python/
 __github_creds__.txt
+
```

Comment:
Can these changes be removed?

Reply:
yes it is removed in the subsequent PR #1782

Reply:
There's no such PR